src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -44,13 +47,16 @@ const roleString = {
   READER: 'Reader',
   WRITER: 'Writer',
   OWNER: 'Owner',
-  PROJECT_OWNER: 'Proj. Owner'
+  PROJECT_OWNER: 'Project Owner'
 }
 
-const InfoTile = ({ title, children }) => {
-  return div({ style: Style.dashboard.infoTile }, [
-    div({ style: Style.dashboard.tinyCaps }, [title]),
-    div({ style: { fontSize: 12 } }, [children])
+const InfoRow = ({ title, subtitle, children }) => {
+  return div({ role: 'row', style: { display: 'flex', justifyContent: 'space-between', margin: '1rem 0.5rem' } }, [
```

Comment:
The rows of key-value pairs might be good place for a description list.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

Reply:
Changed to use description lists in b56fa95b0fc4cef797a80dd80b429799e1f37732